### PR TITLE
Animate zombie

### PR DESCRIPTION
**Pull request by leon-do**
Zombie and text will traverse down and grow.

### CODE REVIEW

File: assets/javascript/game.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+	        $.ajax({
+	            type: "GET",
+	            url: "http://randomword.setgetgo.com/get.php",
+	            dataType: "jsonp",
+	            }).done(RandomWordComplete)
+
+	    }
+	    function RandomWordComplete(data) {
+	    	var randomWord = data.Word;
```
Comment (vbelephants) [inline assets/javascript/game.js]: Make "var randomWord a global variable", you can keep the line here as randomWord = data.Word;

File: assets/javascript/game.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+		var rhymeWord;
+		var letter;
+		var letterIndex = 0;
+		var milliseconds = 5 * 1000;
+
+
+
+
+		generateRandomWord();
+	    function generateRandomWord() {
```
Comment (vbelephants) [inline assets/javascript/game.js]: Can you place all these functions into one object?
Like, wordGenerator = { functions in here }

File: assets/javascript/game.js (hunk 1 above)
Comment (leon-do) [inline assets/javascript/game.js]: I don't want to make too many global variables. That 1 var is only used for that 1 function.

File: assets/javascript/game.js (hunk 2 above)
Comment (leon-do) [inline assets/javascript/game.js]: will leave as is and will put in object later.

**Comment by leon-do** [comment]
Updated the merge conflict. moved my div into the in Game Container: Where the game is played

**Pull request merged**